src/duckling/project/map-select.component.scss

```diff
@@ -1,4 +1,20 @@
+$listContainerWidth: 248px;
+
 md-nav-list {
-    max-height: 60%;
+    max-height: 60vh;
     overflow-y: auto;
+    overflow-x: hidden;
+    width: $listContainerWidth;
+}
+
+md-list-item {
+    width: $listContainerWidth - 16px;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
```

Comment:
You should instead put a div around the text and then place the white-space, text-overflow, and overflow styles on that div. Also the new div would need a min-width: 0, this will make the ellipsis work.